code/api/src/modules/shipments/model.js

```diff
@@ -12,6 +15,19 @@ module.exports = function(sequelize, DataTypes) {
     deliveryDate: {
       type: DataTypes.DATE
     }
+  }, {
+    hooks: {
+      afterCreate: async (shipment) => {
+        let subscriptionId = shipment.dataValues.subscriptionId
+        let subId = await models.Subscription.findOne({ where: subscriptionId })
+        let products = await models.Product.findAll({ where: { crateId: subId.dataValues.crateId } })
+        let index = Math.floor(Math.random() * products.length);
```

Comment:
Our crate will only ship one product for the list of products in the crate. This will randomly find a product to send to the user

Reply:
Sounds good, that makes sense!